### PR TITLE
port pypy plugin to python3

### PR DESCRIPTION
**Pull request by niol**

### CODE REVIEW

**Comment by xrmx** [comment]
Thanks!

File: plugins/pypy/pypy_setup.py [hunk 1]
```diff
@@ -335,15 +336,15 @@ def uwsgi_pypy_paste_loader(config):
...
-        c = os.getcwd() + '/' + c
+    if c[0] != b'/'[0]:
+        c = os.getcwd() + '/' + c.decode()
     try:
-        from logging.config import fileConfig
+        from paste.script.util.logging_config import fileConfig
         fileConfig(c)
     except ImportError:
-        print("PyPy WARNING: unable to load logging.config")
+        print("PyPy WARNING: unable to load paste.script.util.logging_config")
```
Comment (xrmx) [inline plugins/pypy/pypy_setup.py:347]: Is this some old patch that has been forwarded? This have been changed in 71464fbda59c5e3668bcc3fab8c5aa66935cf95c

File: plugins/pypy/pypy_setup.py [hunk 2]
```diff
@@ -335,15 +336,15 @@ def uwsgi_pypy_paste_loader(config):
     """
     global wsgi_application
     c = ffi.string(config)
-    if c.startswith('config:'):
+    if c.startswith(b'config:'):
         c = c[7:]
-    if c[0] != '/':
-        c = os.getcwd() + '/' + c
+    if c[0] != b'/'[0]:
```
Comment (xrmx) [inline plugins/pypy/pypy_setup.py:341]: This [0] is strange

File: plugins/pypy/pypy_setup.py (hunk 1 above)
Comment (niol) [reply plugins/pypy/pypy_setup.py:347]: Yes this is, will review this better and submit fixes in the coming days.

File: plugins/pypy/pypy_setup.py (hunk 2 above)
Comment (niol) [reply plugins/pypy/pypy_setup.py:341]: It seems useless, you're right.

**Pull request merged**